src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenFifteenFour/PopulateMissingSecurityStamps.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Umbraco.Core.Logging;
+using Umbraco.Core.Persistence.SqlSyntax;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenFifteenFour
+{
+    [Migration("7.15.4", 1, Constants.System.UmbracoMigrationName)]
+    public class PopulateMissingSecurityStamps : MigrationBase
+    {
+        public PopulateMissingSecurityStamps(ISqlSyntaxProvider sqlSyntax, ILogger logger) : base(sqlSyntax, logger)
+        {
+        }
+
+        public override void Up()
+        {
+            // A user with a NULL securityStampToken can't log in after v7.8.0
+            Execute.Sql(@"UPDATE umbracoUser SET securityStampToken = NEWID() WHERE securityStampToken IS NULL");
```

Comment:
v7 still supports mysql right? I'd suggest using a c# Guid.NewGuid() here instead of using SQL to create the guid

Reply:
Ah, good catch! 👍

Reply:
Good point. Do we need to do individual updates to give each user a different GUID, or is it safe to let everyone start with the same Guid.NewGuid()?

Reply:
It is safe to have everyone use the same one, it is not used as a public check for anything, it is just a value to indicate the current representation of the user, so that when it ever changes, the system knows that it needs to clear all authn/authz cookies/etc..